<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AscentImageWidget</class>
 <widget class="QWidget" name="AscentImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>718</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="ascentImageLayout" stretch="1,0,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="FileDropFrame" name="imageFrame">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="imageFrameLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="imageErrorCenteringLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="imageErrorGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Image could not be loaded</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="imageErrorGroupBoxLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="imageErrorLabel">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string notr="true">
									  This image file cannot be shown:
									  &lt;FILENAME&gt;
									  Reason: File not found.

									  You can remove the image, replace the file, or mass relocate image files in the whole database.
									  &lt;CAUTION: This text is always replaced from the code.&gt;
									</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="imageErrorButtonLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="imageErrorRemoveButton">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="imageErrorReplaceButton">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Replace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="imageErrorRelocateButton">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Mass relocate</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="imageScrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>250</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="imageScrollAreaWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>511</width>
           <height>258</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="imageScrollAreaLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="photoInfoLayout" stretch="1,4,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="photoIndexLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Photo &lt;n&gt;/&lt;N&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="photoDescriptionLayout" stretch="1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="photoDescriptionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string notr="true">&lt;PhotoDescription&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="photoDescriptionLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Photo description</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="photoDescriptionEditButtonLayout" stretch="1,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="photoInfoSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="editPhotoDescriptionButton">
         <property name="toolTip">
          <string>Edit photo description</string>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="photoControlsLayout" stretch="1,0,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="photoMoveControlsLayout" stretch="0,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="photoMoveControlsBox">
         <property name="title">
          <string>Move current photo</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="photoMoveControlsBoxLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="photoMoveControlsBoxLeftSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="movePhotoLeftButton">
            <property name="toolTip">
             <string>Move current photo towards the front</string>
            </property>
            <property name="text">
             <string notr="true">&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="movePhotoRightButton">
            <property name="toolTip">
             <string>Move current photo towards the back</string>
            </property>
            <property name="text">
             <string notr="true">&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="photoMoveControlsBoxRightSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="photoMoveControlsSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="photoNavigationControlsLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="firstPhotoButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Go to first photo</string>
         </property>
         <property name="text">
          <string notr="true">&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="previousPhotoButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Go to previous photo</string>
         </property>
         <property name="arrowType">
          <enum>Qt::ArrowType::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextPhotoButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Go to next photo</string>
         </property>
         <property name="arrowType">
          <enum>Qt::ArrowType::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lastPhotoButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Go to last photo</string>
         </property>
         <property name="text">
          <string notr="true">&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="photoSlideshowAndAddRemoveControlsLayout" stretch="1,0,1,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="slideshowSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="slideshowBox">
         <property name="title">
          <string>Slideshow</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="slideshowBoxUpperLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QToolButton" name="slideshowStartStopButton">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Start/stop slideshow</string>
              </property>
              <property name="text">
               <string notr="true">&gt;||</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="slideshowIntervalSpinner">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Time between photos in seconds</string>
              </property>
              <property name="suffix">
               <string>s</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="slideshowAutostartCheckbox">
            <property name="toolTip">
             <string>Start slideshow automatically after changing to another ascent</string>
            </property>
            <property name="text">
             <string>Autostart</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="photoAddRemoveControlsSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addPhotosButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Add one or multiple photos</string>
         </property>
         <property name="text">
          <string notr="true">+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removePhotoButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Remove photo</string>
         </property>
         <property name="text">
          <string notr="true">-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileDropFrame</class>
   <extends>QFrame</extends>
   <header>src/viewer/file_drop_frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
